stripe-ui-core/src/main/java/com/stripe/android/uicore/address/AddressRepository.kt

```diff
@@ -51,17 +51,18 @@ class AddressRepository @Inject constructor(
     }
 
     @VisibleForTesting
-    internal fun add(countryCode: String, listElements: List<SectionFieldElement>) {
+    fun add(countryCode: String, listElements: List<SectionFieldElement>) {
```

Comment:
Does this need to have the annotation?

Reply:
Yes, it's still used in `CardBillingAddressElementTest` which is in payments-ui-core